src/main.cpp

```diff
@@ -127,8 +128,27 @@ int main(int argc, char *argv[])
     sodium_init(); // For the auto-updater
 
 #ifdef LOG_TO_FILE
+    QString logFileDir = Settings::getInstance().getSettingsDirPath();
     logFileStream.reset(new QTextStream);
-    logFileFile.reset(new QFile(Settings::getInstance().getSettingsDirPath()+"qtox.log"));
+    logFileFile.reset(new QFile(logFileDir + "qtox.log"));
```

Comment:
You open the file before moving, so if it is already > 1MB, it will be moved while still open, so qTox will be writing in "qtox.log.1".